config/controller/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/eks-controller
-  newTag: v1.0.0
+  newTag: v1.0.1
```

Comment:
This definitely should not have changed unless you were going through the [release process](https://aws-controllers-k8s.github.io/community/docs/contributor-docs/release/) (and this isn't a release PR :) )